core/app/(user)/communities/actions.ts

```diff
@@ -46,237 +47,49 @@ export const createCommunity = defineServerAction(async function createCommunity
-							order: "ff",
-						},
-						{
-							communityId: communityUUID,
-							name: "Published",
-							order: "gg",
-						},
-						{
-							communityId: communityUUID,
-							name: "Shelved",
-							order: "hh",
-						},
-					])
-					.returning("id"),
-				{ communitySlug: slug }
-			).execute();
-
-			const [_, stagesReturn] = await Promise.all([pubTypesPromise, stagesPromise]);
-			const stages = stagesReturn.map((stage) => stage.id);
-
-			const permissionPromise = autoRevalidate(
-				db
-					.with("new_permission", (db) =>
-						db
-							.insertInto("permissions")
-							.values({
-								memberId: member?.id,
-							})
-							.returning("id")
-					)
-					.insertInto("_PermissionToStage")
-					.values((eb) => [
-						{
-							A: eb.selectFrom("new_permission").select("id"),
-							B: stages[0],
-						},
-						{
-							A: eb.selectFrom("new_permission").select("id"),
-							B: stages[1],
-						},
-						{
-							A: eb.selectFrom("new_permission").select("id"),
-							B: stages[2],
-						},
-						{
-							A: eb.selectFrom("new_permission").select("id"),
-							B: stages[3],
-						},
-					]),
-				{ communitySlug: slug }
-			).execute();
-
-			const moveConstraintPromise = autoRevalidate(
-				db.insertInto("move_constraint").values([
-					{
-						//  Submitted can be moved to: Consent, To Evaluate, Under Evaluation, Shelved
-						stageId: stages[0],
-						destinationId: stages[1],
-					},
-					{
-						stageId: stages[1],
-						destinationId: stages[2],
-					},
-					{
-						stageId: stages[2],
-						destinationId: stages[3],
-					},
-					{
-						stageId: stages[3],
-						destinationId: stages[4],
-					},
-					{
-						stageId: stages[4],
-						destinationId: stages[5],
-					},
-				]),
-				{ communitySlug: slug }
-			).execute();
-
-			const createPubPromise = autoRevalidate(
-				db
-					.with("new_pubs", (db) =>
-						db
-							.insertInto("pubs")
-							.values({
-								communityId: communityUUID,
-								pubTypeId: pubTypeId as PubTypesId,
-							})
-							.returning("id")
-					)
-					.with("pubs_in_stages", (db) =>
-						db.insertInto("PubsInStages").values((eb) => [
-							{
-								pubId: eb.selectFrom("new_pubs").select("id"),
-								stageId: stages[0],
-							},
-						])
-					)
-					.insertInto("pub_values")
-					.values((eb) => [
-						{
-							pubId: eb.selectFrom("new_pubs").select("new_pubs.id"),
-							fieldId: fields.find((field) => field.slug === `${slug}:title`)!.id,
-							value: '"The Activity of Slugs I. The Induction of Activity by Changing Temperatures"',
-						},
-						{
-							pubId: eb.selectFrom("new_pubs").select("new_pubs.id"),
-							fieldId: fields.find((field) => field.slug === `${slug}:content`)!.id,
-							value: '"LONG LIVE THE SLUGS"',
-						},
-					]),
-				{ communitySlug: slug }
-			).execute();
-			await Promise.all([permissionPromise, moveConstraintPromise, createPubPromise]);
-		}
```

Comment:
Adding all this seed data made sense when we had no way to create a lot of these things in the app. But now that we're actually using the button in prod it's kind of annoying, especially because some of the things created here can't be deleted. So I removed it for now, with the assumption that we'll build a few specific templates of seed data into this page in the future.

That does slow down the E2E tests a bit, but I also think it's good for them to exercise the pub/field/type/stage crud interfaces